contributed_definitions/NXevent_data_em.nxdl.xml

```diff
@@ -0,0 +1,168 @@
+         session. Subsequently, data (images, spectra, etc.) can be collected. 
+         Users may wish to take a single data item after which their session is complete.
+         
+         For this use case one may argue that the need for an additional grouping 
+         is redundant. Instead, the metadata could equally have been stored inside
+         the respective groups of the top-level mandatory NXinstrument group. 
+         On the flip side, even for a session with a single image it would also not 
+         harm to nest the data.
+         
+         Frequently users spend much longer at the microscope in the course of which
+         the recalibrate and take data items with each calibration.
+         
+         In fact, oftentimes scientists feel that there is a need to store details 
+         about eventual drift of the specimen in its holder (if such data is available)
+         or record changes the lens excitations or apertures used. 
+         Although, current microscopes are usually equipped with stabilization 
+         systems for many of the individual components, it can still be useful 
+         to store time-dependent data more seriously.
+         
+         Another reason if not a need for a more finely granularizable options for 
+         storing time-dependent data, is that over the course of a session one may
+         reconfigure the microscope. What is a reconfiguration? This could be the
+         change of an aperture mode because a scientist may first collect an image
+         with some aperture and then choose a different mode before taking the next
+         image. As the aperture affects the electron beam it will affect the system.
+         Not necessarily to a level that the microscopy requires a recalibration but
+         at least at the level of each image the application definition has to assure
+         that these specific settings can be stored for each collected signal/image.
+         
+         Let aside for a moment the technology and business models, an EM could be 
+         monitored (and will likely be in the future) for streaming out spatio-temporal
+         details about its components, locations and eventually applied stimuli and 
+         positioning of the specimen. Some snapshot or integrated data from this stream
+         are relevant to understand the signal genesis. In such a generic case
+         it might be necessary to sync these streaming data with those intervals
+         in time when specific measurements are taken (spectra collected, 
+         images taken, diffraction images indexed on-the-fly).
+         
+         Theoretically, an instrument and specimen should be considered as dynamic.
+         Scientists often report or feel (difficult to quantify) observations that
+         microscopes &quot;&quot;&quot;perform differently&quot;&quot;&quot; across sessions, without sometimes a 
```

Comment:
quotation marks need attention here

Reply:
formatting fixed, will carry this as mentioned also through for the other places where bullet point lists when havoc, thank you prjemian